modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/components/PartialResults.tsx

```diff
@@ -29,7 +29,21 @@ const PartialResults: React.FC<IProps> = ({
 	onShow,
 	reportDataURL,
 }) => {
-	const {resource} = useResource({fetch, link: reportDataURL});
+	const [resourceState, setResourceState] = useState(() => ({
+		error: false,
+		loading: true,
+	}));
+	const {error, loading} = resourceState;
```

Comment:
Since the resourceState is not being used anywhere else we can just use destructuring to simplify this code:
```suggestion
	const [{error, loading}, setResourceState] = useState(() => ({
		error: false,
		loading: true,
	}));
```